scripts/roll-handler.js

```diff
@@ -449,7 +449,9 @@ Hooks.once('tokenActionHudCoreApiReady', async (coreModule) => {
          */
         #rollItem (event, actor, actionId) {
             const item = actor.items.get(actionId)
-
+            if (item.system?.selfEffect) {
+                createSelfEffectMessage(item);
```

Comment:
Unfotunately, `createSelfEffectMessage` throws an error as it's not defined. Is it defined somewhere else? I think this was the issue I had when I looked as it didn't seem to be available in a publically-defined function, only within the event listener.